MyClassroom.Application/Services/IUserService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MyClassroom.Core.Models;
+
+namespace MyClassroom.Application.Services
+{
+    public interface IUserService
+    {
+        Task<User?> AddRole(int userId, int roleId);
+        Task<User?> AddRole(int userId, string roleName);
+        Task<User> AddUser(User user);
+        Task<User?> GetUserByIdAsync(int id);
```

Comment:
Let's try to bo consistent. If you want to use Async postfix please use it in all the methods.

Reply:
Yes, I will change them